src/modules/test/render.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @jest-environment jsdom
+ */
+
```

Comment:
- [ ] It would be better if we can successfully run the test suite. However, I encountered what I think is a node modules problem as seen in the screenshot below
![Screenshot from 2022-11-25 16-25-05](https://user-images.githubusercontent.com/46388113/204027238-631df6ae-cef0-45f0-b285-b81adfd54561.png)

Here is what I did in trying to resolve it, I removed the node_modules and package-lock.json files. Did the `npm i` again then after `npm run test`. All these couldn't resolve it

Reply:
@addod19 Thank you very much for your review, now you can run **npm run test** and it will work 👯